src/python/aqueduct_executor/operators/connectors/tabular/main.py

```diff
@@ -117,54 +125,57 @@ def setup_connector(
             PostgresConnector as OpConnector,
         )
     elif connector_name == common.Name.SNOWFLAKE:
-        from aqueduct_executor.operators.connectors.tabular.snowflake import (
+        from aqueduct_executor.operators.connectors.tabular.snowflake import (  # type: ignore
```

Comment:
Will all these type ignores be fixed by ENG-1285?

Reply:
No this is another issue where mypy can't deal with case-based imports. I think ignoring mypy here is the right thing since our use case is a bit special.